CDTDatastore/CDTReplicator/CDTPullReplication.h

```diff
@@ -48,6 +52,9 @@
 
 /** All CDTPullReplication objects must have a source and target.
 
+  This is the same as calling - replicationWithSource:target:username:password with `nil` set
```

Comment:
I would remove the dash and fix up the code i.e. `replicationWithSource:source target:target username:nil password:nil`.